### PR TITLE
Graphana // External Formatters [WIP]

### PR DESCRIPTION
**Pull request by amarjandu**
puts notifications into DSS-{STAGE} Namespace for cloudwatch metrics.

### CODE REVIEW

File: chalice/app.py [hunk 1]
```diff
@@ -8,33 +8,34 @@
...
 import monitor
+from monitor import events
 
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
-stage = os.getenv('DSS_INFRA_TAG_STAGE')
-app = chalice.Chalice(app_name=f"dss-monitor-{stage}")
+app = chalice.Chalice(app_name=f"dss-monitor-dev")
```
Comment (xbrianh) [inline chalice/app.py]: I think chalice will attach the stage suffix on deployment, so `dss-monitor` is sufficient here.

File: monitor/events/__init__.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import os
+import datetime
+from dcplib.aws.clients import cloudwatch  # noqa
+
+
+def ms_since_epoch(start_time=None):
```
Comment (xbrianh) [inline monitor/events/__init__.py]: Is this used anywhere?

File: monitor/events/__init__.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    """ :returns: returns datetime object of time since epoch"""
+    epoch = datetime.datetime.utcfromtimestamp(0)
+    if start_time is None:
+        start_time = datetime.datetime.utcnow()
+    return start_time - epoch
+
+def putMetrics(namespace: str = None, metric_name: str = None, timestamp=datetime.datetime.utcnow(),
```
[the remark this answers is not in the document]
Comment (amarjandu) [reply monitor/events/__init__.py]: removed deafult to None

File: README.md [hunk 4]
```diff
@@ -34,5 +34,8 @@ The DSS-Monitor can provide notifications for daily progress in a slack channel.
 
 Once the appropriate service account config file has been saved according the instructions above, and there has been a
 webhook placed into the secrete manager, perform:
-`make DEPLOYMENT={STAGE} deploy`
-to deploy the lambdas, and subscribe to the appropriate notifications in the DSS.
\ No newline at end of file
+```
+make deploy
```
[the remark this answers is not in the document]
Comment (amarjandu) [reply README.md]: resolved

File: README.md [hunk 5]
```diff
@@ -34,5 +34,8 @@ The DSS-Monitor can provide notifications for daily progress in a slack channel.
 
 Once the appropriate service account config file has been saved according the instructions above, and there has been a
 webhook placed into the secrete manager, perform:
```
[the remark this answers is not in the document]
Comment (amarjandu) [reply README.md]: resolved

**Pull request merged**